src/performance/osu/OsuScore.cpp

```diff
@@ -70,8 +70,9 @@ void OsuScore::computeTotalValue()
 	if ((_mods & EMods::NoFail) > 0)
 		multiplier *= 0.90f;
 
+	int numTotalHits = TotalHits();
 	if ((_mods & EMods::SpunOut) > 0)
-		multiplier *= 0.95f;
+		multiplier *= 1.0f - std::pow(beatmap.NumSpinners() / static_cast<f32>(numTotalHits), 0.85f);
```

Comment:
Is this really doing much? It's basically almost a linear function:
![image](https://user-images.githubusercontent.com/1329837/101484760-79244180-399d-11eb-9269-fde32bd9a8e0.png)

With the delta being ~0.06 at most. Do you have anything against making this just the following for simplicity?

```suggestion
		multiplier *= 1.0f - beatmap.NumSpinners() / static_cast<f32>(numTotalHits);
```

Reply:
I believe it was to make it closer to current 0.95 multiplier for maps with enough spinners

Reply:
Sure, I guess it's not a big deal...